service/src/test/kotlin/app/cash/backfila/service/LeaseHunterTest.kt

```diff
@@ -48,8 +48,12 @@ class LeaseHunterTest {
           .build())
     }
     scope.fakeCaller(user = "molly") {
-      createBackfillAction.create("deep-fryer",
-          CreateBackfillRequest("ChickenSandwich"))
+      createBackfillAction.create(
+          "deep-fryer",
+          CreateBackfillRequest.Builder()
+              .backfill_name("ChickenSandwich")
```

Comment:
Should it be Builder("ChickenSandwich")?

Reply:
don't think you can do that